src/coreclr/jit/llvm.cpp

```diff
@@ -2123,6 +2105,15 @@ void Llvm::ConvertShadowStackLocalNode(GenTreeLclVarCommon* node)
             case GT_LCL_VAR:
                 indirOper = lclVar->TypeIs(TYP_STRUCT) ? GT_OBJ : GT_IND;
                 break;
+            case GT_LCL_FLD:
+                // TODO-LLVM: TYP_STRUCT
+                if (lclVar->TypeIs(TYP_STRUCT))
+                {
+                    failFunctionCompilation();
+                }
+
+                indirOper = GT_IND;
+                break;
```

Comment:
```suggestion
                assert(!lclVar->TypeIs(TYP_STRUCT));
                indirOper = GT_IND;
                break;
```
It's not that we don't have an implementation for it in the LLVM backend, it's that it cannot exist in the compiler at all (with some exceptions that should not be relevant here).

Reply:
ok, thanks